opennlp-dl/src/main/java/opennlp/dl/doccat/DocumentCategorizerConfig.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package opennlp.dl.doccat;
+
+import java.util.Map;
+
+public class DocumentCategorizerConfig {
+
+  private Map<String, String> id2label;
+
+  public Map<String, String> getId2label() {
+    return id2label;
```

Comment:
it appears to be loaded and then read only, without modifying the map. Maybe this could return a read-only map? To avoid somewhere else we accidentally modify it unintentionally (when that happens we can review and check if we need to worry about threads...)